NIMATRO/NIMATRO-IOC-01App/Db/controls.db

```diff
@@ -0,0 +1,422 @@
+record(bo, "$(P)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+}
+
+record(bo, "$(P)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+
+# Initially generated by $Id$
+# 
+# auto-generated EPICS records specify an asyn port "lvfp", but this can be changed - it just needs
+# to match the first argument of the relevant lvDCOMConfigure() command in the IOC st.cmd 
+# 
+      
+# 
+# Definitions from configSection "frontpanel" in XML input file
+#
+#     lvDCOMConfigure("lvfp", "frontpanel", "/path/to/lvinput.xml")
+# 
+
+# Write to LabVIEW control "Speed" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)SPEED")
+{
+	field(DESC, "LabVIEW 'Speed'")
```

Comment:
The auto-generated descriptions aren't very useful. Comment repeats as necessary.

Reply:
Updated description fields for all records.